packages/image_picker/image_picker/README.md

```diff
@@ -42,22 +43,20 @@ If you require your picked image to be stored permanently, it is your responsibi
 
 ### Example
 
+<?code-excerpt "readme_excerpts.dart (Pick)"?>
 ``` dart
-import 'package:image_picker/image_picker.dart';
-
-    ...
-    final ImagePicker _picker = ImagePicker();
-    // Pick an image
-    final XFile? image = await _picker.pickImage(source: ImageSource.gallery);
-    // Capture a photo
-    final XFile? photo = await _picker.pickImage(source: ImageSource.camera);
-    // Pick a video
-    final XFile? image = await _picker.pickVideo(source: ImageSource.gallery);
-    // Capture a video
-    final XFile? video = await _picker.pickVideo(source: ImageSource.camera);
-    // Pick multiple images
-    final List<XFile>? images = await _picker.pickMultiImage();
-    ...
+final ImagePicker picker = ImagePicker();
+// Pick an image.
+final XFile? image = await picker.pickImage(source: ImageSource.gallery);
+// Capture a photo.
+final XFile? photo = await picker.pickImage(source: ImageSource.camera);
+// Pick a video.
+final XFile? galleryVideo =
```

Comment:
The variable names changed because there were variable redeclarations that didn't compile.